app/geo/gui/src/ts/twxplore/gui/geo/states/map/ROOT_FEATURE_URI.ts

```diff
@@ -0,0 +1,3 @@
+type ROOT_FEATURE_URI = "http://twxplore.github.io/app/geo/feature#root";
+export const ROOT_FEATURE_URI: ROOT_FEATURE_URI =
```

Comment:
You don't really need the type, that's mostly for action names where you want to do a switch.